### DISCUSSION TITLE
Some newbie questions

### DISCUSSION THREAD

**Discussion by Ud4ba** [lxqt/lxqt#2265] [Q&A]
Hello everyone . Thanks for this wonderful and lightweight project. Im new here. I have some questions if someone can help me.

1)How i can set the background color of PCManFM-QT at dark?
2)Is there any way i can add more sensors and be able to see temperature of CPU+GPU+Disks on Panel? Sensors widget only recognizes coretemp-isa-000 which is the CPU temps only.
3)Getting this error while trying to enable fstrim.service
 : The unit files have no installation config (WantedBy=, RequiredBy=, Also=,
Alias= settings in the [Install] section, and DefaultInstance= for template
units). This means they are not meant to be enabled using systemctl.
 
Possible reasons for having this kind of units are:
• A unit may be statically enabled by being symlinked from another unit's
  .wants/ or .requires/ directory.
• A unit's purpose may be to act as a helper for some other unit which has
  a requirement dependency on it.
• A unit may be started when needed via activation (socket, path, timer,
  D-Bus, udev, scripted systemctl call, ...).
• In case of template units, the unit is meant to be enabled with some
  instance name specified.

4)How i can edit the sign in page at boot up?

5There is a problem with tittles at windows.Whem setting to not show tittles Firefox will show 2 tittles (with 2 close,minimize,etc) Any workaround for this?

**Comment by stefonarch** [comment]
Be welcome! You didn't tell us OS and LXQt version, in older versions some features are not present.

1) Appearance > Palette
2) `lm-sensors` is installed? You can also use [scripts](https://github.com/lxqt/lxqt/discussions/2179) in "custom command"
3) Not related to LXQt, afaik you need to activate only the timer which will do the rest: `sudo systemctl enable fstrim.timer `
4) By installingor editing SDDM themes. For arch and derivatives we recommend https://aur.archlinux.org/packages/sddm-conf-git, other documentation can be found in the arch SDDM wiki page.
5) I didn't understand where you set this. Window titles are controlled by the WM, which one do you use?

**Comment by Ud4ba** [comment]
Im sorry about it . Im using Debian testing with 0.16.0 version.
1.Really thanks!
2.lm-sensors is installed yes.How i can add a script for that? nvidiagpu temprature and CPU ? 
edit:sensors detect wont add anything new to sensors. On previous install using Gnome i could see hdd.ssd.cpu and gpu.
3.Im sorry for that,i though it the time i was writting it down.I did exactly that and again getting the same error "Your systemd unit doesn't have an [Install] section. It does, however, have an [INSTALL] section. Section names are case sensitive."  What is workaround for this? If you want to provide me support here , i will find out .
4)Im using Debian testing. Havent done this before. I would study arch wiki for a while before asking further assistance. Will this do the job? : https://github.com/3ximus/aerial-sddm-theme If im using Debian its like Kubuntu or Lubuntu?
5)I solved it somehow . It was a weird bug that was showing twice the tittle of firefox window. (minimize option , fullscreen, close etc)

**Reply by stefonarch** [reply]
> Im using Debian testing with 0.16.0 version.

I'd suggest to update LXQt: https://lxqt-project.org/blog/2022/05/06/latest-lxqt-debian-ubuntu/  - 0.16 is really outdated, no palettes, no custom command plugin etc.

Arch wiki is suitable for most things and the most completed doc around. As *ubuntu is debian based most things are similar.

**Reply by Ud4ba** [reply]
Thanks again. Didnt know that existed.
If the packages upgraded at debian repos will be there any problems or actions needed ?
